Source/missiles.cpp

```diff
@@ -4146,7 +4150,11 @@ void ProcessMissiles()
 
 	i = 0;
 	while (i < nummissiles) {
+#ifdef HELLFIRE
+		if (missile[missileactive[i]]._miDelFlag == TRUE) {
+#else
 		if (missile[missileactive[i]]._miDelFlag) {

```

Comment:
can you check if != FALSE works for both

Reply:
it doesn't

Reply:
I wonder if maybe _miDelFlag was changed from `BOOL` to `bool` and that's the difference?

Reply:
well, the asm in hellfire says `cmp dword ptr [eax+0x697A8C], 0x01`

Reply:
What an ugly and useless refactor :D

Reply:
indeed :P